virttest/utils_v2v.py

```diff
@@ -1304,8 +1304,16 @@ def _v2v_post_cmd():
             for i in range(int(params.get('_disk_count', 0))):
                 options += ' -oo rhv-disk-uuid=%s' % str(uuid.uuid4())
 
+        # Protect the blanks in original guest name
+        safe_vm_name = ''
+        if ' ' in vm_name and vm_name in options:
+            safe_vm_name = vm_name.replace(' ', '%20')
+            options = options.replace(vm_name, safe_vm_name)
         # Construct a final virt-v2v command and remove redundant blanks
         cmd = ' '.join(('%s %s' % (V2V_EXEC, options)).split())
+        # Restore the real original guest name
+        if safe_vm_name:
+            cmd = cmd.replace(safe_vm_name, vm_name)
```

Comment:
Oh, I see this replacing is a temporary move. Then I would recommend an simpler string which could be understood better than '%20' which seems unnecessary since we're not trying to use it in a url. Just my opinion

Reply:
I don't think it will bring readability problem here, as you said it just a temp replacement, users could understand it by the context and code comments, and also '%20' is a famous replacement in URL, we could borrow the idea from it.

Reply:
Anyway, I did a bit change about this.

Reply:
It's quite understandable.